action.yml

```diff
@@ -103,4 +107,5 @@ runs:
           --thread-comments=${{ inputs.thread-comments }} \
           --ignore="${{ inputs.ignore }}" \
           --database=${{ inputs.database }} \
-          --file-annotations=${{ inputs.file-annotations }}
+          --file-annotations=${{ inputs.file-annotations }} \
+          --extra-arg=${{ inputs.extra-arg }}
```

Comment:
Should be in quotes since the only way to support multiple extra args in this action will be using spaces.
```yml
- uses: cpp-linter/cpp-linter-action@v2
  with:
    extra-args: "-std=c++14 -Wall"
```
Notice that using `-std c++14` (without using an `=`) should be discouraged because some OSs don't allow the arg name and value to be separated by a space.